includes/ucf-news-config.php

```diff
@@ -6,16 +6,19 @@
 	class UCF_News_Config {
 		public static
 			$default_options = array(
-				'title'    => 'News',
-				'layout'   => 'classic',
-				'sections' => '',
-				'topics'   => '',
-				'limit'    => 3
+				'title'          => 'News',
+				'layout'         => 'classic',
+				'sections'       => '',
+				'topics'         => '',
+				'limit'          => 3,
+				'perrow'         => 3
```

Comment:
`perrow` would be a good option to assign intval formatting to in UCF_News_Config::format_options()